Sources/CohoSDK/CohoSDK.swift

```diff
@@ -117,6 +107,18 @@ public class CohoSDK {
         }.resume()
     }
 
+    private func createRequest() -> URLRequest {
+        var request = URLRequest(url: endpoint)
+        request.httpMethod = "POST"
+        request.setValue(Constants.Headers.applicationJson, forHTTPHeaderField: Constants.Headers.contentType)
+        request.setValue(Constants.Headers.acceptAll, forHTTPHeaderField: Constants.Headers.accept)
+        request.setValue(tenantId, forHTTPHeaderField: Constants.Headers.tenantIdKey)
+        request.setValue(Constants.Headers.userId, forHTTPHeaderField: Constants.Headers.userIdKey)
+        request.setValue("sdk", forHTTPHeaderField: Constants.Headers.dataContext)
```

Comment:
When creating a mapping the default value of datSourceContext is "default". Here we always pass the value "sdk".
Is this intentional?

Reply:
Indeed, 
We want to identify events fired from the sdk to be able to automap (at least) all the values which we are extracting implicit from the client device